core/persist/viewmanager/ViewManagerModel.ts

```diff
@@ -87,247 +97,218 @@ export class ViewManagerModel<T extends PlainObject = PlainObject>
      */
     static async createAsync(config: ViewManagerConfig): Promise<ViewManagerModel> {
         const ret = new ViewManagerModel(config);
-        await ret.loadAsync();
+        await ret.initAsync();
         return ret;
     }
 
     /** Immutable configuration for this model. */
     readonly viewType: string;
-    readonly displayName: string; // from viewTypeDisplayName, or generated off of viewType
-    readonly DisplayName: string; // capitalized viewTypeDisplayName
+    readonly typeDisplayName: string;
+    readonly globalDisplayName: string;
     readonly enableDefault: boolean;
-    readonly enableAutoSave: boolean;
     readonly enableFavorites: boolean;
-
-    /** Last selected, fully-persisted state of the active view. */
-    @observable.ref value: T = {} as T;
-    /** Current state of the active view, can include not-yet-persisted changes. */
-    @observable.ref pendingValue: T = {} as T;
-    /** Loaded saved view definitions - both private and shared. */
-    @observable.ref views: View<T>[] = null;
-
-    /** Currently selected view, or null if in default mode. Token only will be set during pre-loading.*/
-    @observable selectedToken: string = null;
-    @observable.ref selectedView: View<T> = null;
-
+    readonly manageGlobal: boolean;
+
+    /** Current view or the "default" view. Will not include uncommitted changes */
+    @observable.ref view: View<T> = View.DEFAULT;
+    /** Pending (transient) view, can include not-yet-saved changes. */
+    @observable.ref pendingView: View<T> = this.view;
+    /** Loaded saved view library - both private and global */
+    @observable.ref views: ViewInfo[] = [];
```

Comment:
No doubt also something you discussed, but wish we didn't have the singular property be one type and the plural of the same name be the other...  `viewInfos` pretty wonky though, eh?